jiant/utils/python/filesystem.py

```diff
@@ -14,3 +14,30 @@ def find_files(base_path, func):
 
 def find_files_with_ext(base_path, ext):
     return find_files(base_path=base_path, func=lambda filename: filename.endswith(f".{ext}"))
+
+
+def get_code_base_path():
+    """Gets path to root of jiant code base
+
+    Returns:
+        Path to root of jiant code base
+    """
+    return os.path.abspath(os.path.join(__file__, "../../../.."))
```

Comment:
Best to use os.path.dirname or equivalent instead of string

Reply:
Using os.pardir instead